src/app/[locale]/auth/reset/form.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable import/no-extraneous-dependencies */
+"use client";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useFormState, useFormStatus } from "react-dom";
+import { useForm, type UseFormRegister } from "react-hook-form";
+import { Button, Input } from "@/components/uikit";
+import {
+  resetFormSchema,
+  resetPassword,
+  type ResetPasswordState,
+  type ResetFormInput,
+} from "@/lib/actions/auth";
+import { css } from "~/styled-system/css";
+import { VStack } from "~/styled-system/jsx";
+
+function ResetFormInner({
+  register,
+  isValid,
+}: {
+  register: UseFormRegister<ResetFormInput>;
+  isValid: boolean;
+}) {
+  const { pending } = useFormStatus();
+
+  return (
+    <VStack gap="15px">
+      <Input
+        variant="outline"
+        placeholder="Введите email"
+        {...register("email")}
+      />
+      <p className={css({ textStyle: "body.3", color: "neutral.2" })}>
+        Введите адрес электронной почты используемый для входа и мы вышлем вам
+        письмо со ссылкой для создания нового пароля
```

Comment:
момент по тексту, давай сразу его в интл обертку:
```
import { useTranslations } from "next-intl";
const t = useTranslations();
```
```suggestion
        t("auth.reset_form_helper") 
```